### PR TITLE
Marcus/fix210122 2

### PR DESCRIPTION
**Pull request by MarcusKlapprodt**
changed the top padding

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -230,10 +230,11 @@ a:visited {
  * CONTENT
  */
 
-/* .content {
-  background: #fff;
-  padding: 20px;
-} */
+.content {
+  /* background: #fff; */
+  padding: 4em 0 0 0;
```
Comment (fstreicher) [inline assets/style.css:235]: please use `margin-top`

**Review by fstreicher — CHANGES_REQUESTED** [review]
see comment

File: assets/style.css (hunk 1 above)
Comment (MarcusKlapprodt) [reply assets/style.css:235]: @fstreicher okay - how do I proceed now with this open branch? Close and open a new one with margin or can I resolve it directly here?

**Review by fstreicher — APPROVED** [review]
LGTM 👍

**Pull request merged**